app/helpers/application_helper.rb

```diff
@@ -74,4 +74,8 @@ def html_title(module_item)
   def calculate_module_state
     CalculateModuleState.new(user: current_user).call
   end
+
+  def footer_static_pages
+    Contentful::Static.find_by(addToFooter: true).load
```

Comment:
We should try and use snake_case on our attributes, but we could also simplifiy the name. Why not just `footer`.

Reply:
agreed `footer`

Reply:
Done